spec/cookbook_spec.rb

```diff
@@ -167,21 +207,76 @@
           :path => 'OWNERS',
         },
       ],
+      :result => [],
+      :result_with_symlink_tracking => [],
+    },
+    {
+      :name => 'when metadata file is not in the root of the cb dir',
+      :files => [
+        {
+          :status => :deleted,
+          :path => 'cookbooks/one/cb_one/files/default/metadata.rb',
+        },
+      ],
       :result => [
+        ['cb_one', :modified],
       ],
+      :result_with_symlink_tracking => [
+        ['cb_one', :modified],
+        ['cb_one', :modified],
+      ],
+    },
+    {
```

Comment:
We should have a comment here that this is the real test around symlinks.

Reply:
They are all valid tests.